### PR TITLE
Documentation: add instruction on vendoring, build

### PR DESCRIPTION
**Pull request by gyuho**
Address https://github.com/coreos/etcd/issues/5857#issuecomment-230174840

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,11 +41,15 @@ The easiest way to get etcd is to use one of the pre-built release binaries whic
 
 For those wanting to try the very latest version, you can build the latest version of etcd from the `master` branch.
 You will first need [*Go*](https://golang.org/) installed on your machine (version 1.6+ is required).
+etcd vendors its dependency for official release binaries, while making vendoring optional to avoid import conflicts.
```
Comment (xiang90) [inline README.md]: move this to https://github.com/coreos/etcd/blob/master/Documentation/dl_build.md#build-the-latest-version?

Also put a link here to the dl_build.md page instead?

Comment (gyuho) [inline README.md]: Ok will do. thanks.

**Comment by gyuho** [comment]
@xiang90 PTAL.

Thanks!

**Comment by xiang90** [comment]
lgtm

**Pull request merged**